### PR TITLE
Increase bitSize when parsing the SIZE attribute of a mets file

### PR DESCRIPTION
**Pull request by gtpho**
Deze change fixt het volgende:

> Fouten Kibana:
>
> Nov 8, 2022 @ 13:47:36.042 2.25.119 A.11ED - OAI-PMH - error oai-pmh unable to process mets header strconv.ParseInt: > parsing "14924267520": value out of range
> Nov 8, 2022 @ 13:47:36.042 2.25.119 A.11ED.1 d50f6a44-73a7-41e8-a4f5-d38d5904ed21 OAI-PMH 0 error unable to publish finding aid strconv.ParseInt: parsing "14541513728": value out of range

### CODE REVIEW

File: hub3/ead/mets.go [hunk 1]
```diff
@@ -137,7 +137,7 @@ func updateFileInfo(files map[string]*eadpb.File, fg []*CfileGrp, fa *eadpb.Find
 		file.MimeType = metsFile.AttrMIMETYPE
 		fa.GetMimeTypes()[file.MimeType]++
 
-		size, err := strconv.ParseInt(metsFile.AttrSIZE, 10, 32)
+		size, err := strconv.ParseInt(metsFile.AttrSIZE, 10, 64)
```
Comment (gtpho) [inline hub3/ead/mets.go:140]: @kiivihal @Bram74  Ik kan niet op de warning hieronder (regel 145) een comment zetten want het is geen diff dus doe ik het via hier. Weten een van jullie toevallig waar deze protobuf bestand allemaal gebruikt wordt?

[the remark this answers is not in the document]
Comment (kiivihal) [reply hub3/ead/mets.go:140]: staticcheck flags strconv.Atoi as a securityrisk, and prefer to use strconv.ParseInt.

Comment (kiivihal) [reply hub3/ead/mets.go:140]: There were some issue with PHP or Javascript protobuf clients having difficulty with int64 but I can't remember exactly. 

I think in this case it is better to change it to int64 in the protobuf file.

Comment (gtpho) [reply hub3/ead/mets.go:140]: @kiivihal Indeed but changing it in the protobuf file means we need to regenerate it for every client that uses it right? I'm trying to figure out which clients are using it.

**Pull request merged**